### PR TITLE
Clb697

### PR DESCRIPTION
**Pull request by dereknoblej**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 113 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-9ce70eb20/docs/cloud-load-balancers/v1/). valid for 14 days

File: api-docs/cloud-load-balancers-v1/api-reference/methods/get-show-current-usage-v1.0-account-loadbalancers-loadbalancerid-usage-current.rst [hunk 1]
```diff
@@ -112,7 +88,7 @@ Response
 
     {
         "links": [],
-        "loadBalancerUsageRecords": [
+        "loadBalancerUsageRecord": [
```
Comment (the2hill) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/get-show-current-usage-v1.0-account-loadbalancers-loadbalancerid-usage-current.rst]: Both JSON and XML responses seem to be incorrect here. 
Example JSON response I'm seeing. 
`{
    "loadBalancerUsageRecords": [
        {
            "averageNumConnections": 0,
            "averageNumConnectionsSsl": 0,
            "numPolls": 12,
            "sslMode": "ON",
            "outgoingTransfer": 0,
            "incomingTransfer": 0,
            "outgoingTransferSsl": 0,
            "incomingTransferSsl": 0,
            "numVips": 1,
            "endTime": "2019-12-04T21:00:00Z",
            "vipType": "PUBLIC",
            "startTime": "2019-12-04T20:00:00Z",
            "id": 10999135
        },....`

File: api-docs/cloud-load-balancers-v1/api-reference/methods/get-show-current-usage-v1.0-account-loadbalancers-loadbalancerid-usage-current.rst [hunk 2]
```diff
@@ -135,19 +112,8 @@ Response
...
-    		<outgoingTransferSsl>0</outgoingTransferSsl>
-    		<incomingTransferSsl>0</incomingTransferSsl>
-    		<numVips>1</numVips>
-    		<vipType>PUBLIC</vipType>
-    		<averageNumConnections>0</averageNumConnections>
-    		<averageNumConnectionsSsl>0</averageNumConnectionsSsl>
-    		<numPolls>0</numPolls>
-    		<sslMode>OFF</sslMode>
-    	</loadBalancerUsageRecords>
+      	<loadBalancerUsageRecord id="42030" averageNumConnections="0.0" incomingTransfer="0" outgoingTransfer="0"
```
Comment (the2hill) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/get-show-current-usage-v1.0-account-loadbalancers-loadbalancerid-usage-current.rst]: This doesn't seem to match current xml. Please reverify

**Comment by rackernexus** [comment]
Build 114 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-71193de2b/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by the2hill** [comment]
Changes here seem good. But think we were going to still update the paragraph to account for required params

**Comment by rackernexus** [comment]
Build 115 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-09b842332/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by the2hill** [comment]
LGTM +2

**Comment by cyrichardson** [comment]
@dereknoblej @the2hill Merged

**Pull request merged**